signature/policy_eval.go

```diff
@@ -283,7 +283,7 @@ func (pc *PolicyContext) IsRunningImageAllowed(ctx context.Context, publicImage
 		allowed, err := req.isRunningImageAllowed(ctx, image)
 		if !allowed {
 			logrus.Debugf("Requirement %d: denied, done", reqNumber)
-			return false, err
+			return false, PolicyRequirementError(err.Error())
```

Comment:
No; `PolicyRequirementError` means “the policy says you can’t do that”, which is not _all_ errors (e.g. “fetching signatures failed because the network is down” is explicitly not a `PolicyRequirementError`).

Maybe the documentation of that type needs to be improved.

Reply:
Makes sense, thank you for the clarification.